chore(clerk-js,types): Convert Saml related types from experimental to stable (#1876)
clerkinc/javascript: .changeset/happy-turkeys-explode.md

---
'@clerk/clerk-js': patch
'@clerk/types': patch
---

> Make SAML related types from experimental to stable:

Reviewer: ```suggestion
Mark the following SAML related types as stable:
```
